reference/api/instore_orders.yaml

```
@@ -314,7 +314,7 @@ components:
   schemas:
     currency_id:
       type: string
-      example: ARS
+      example: COP
```

```suggestion
      example: ARS
```
Esta documentación no puede localizarse, por lo cual ponemos como ejemplo el currency_id de un site con mas share, como MLA (aunque podría ser de MLB también)

El valor para Colombia se especifica más abajo en toda la lista de valores posibles (enum)

---

Nosotros decidimos cambiar a COP el ejemplo debido a que es el unico site con esta version activa, los demas sites no tienen esta version de la api, todas se actualizaron ya. si aun asi, no lo ven viable, lo dejamos en ARS

---

Hola @raul-crisostomo ! ¿Cómo es que esta versión está solo activa en Colombia? Los cambios que se sugieren aquí serían visibles para todos los sites, no solo para Colombia. No me queda claro esto. ¡Gracias!

---

Esa version de producto solo esta productivo en colombia @abderossi a pesar de ser visibles en todos los sites, solo colombia usa "orders" v1